cypress/fixtures/suite/backoffice/ssp-model-management/static-ssp-model-management.json

```diff
@@ -0,0 +1,8 @@
+{
+  "defaultPassword": "change123",
+  "sspModel": {
+    "name": "Sample Model Name",
+    "code": "sample-model-code",
+    "image": "suite/backoffice/ssp-model-management/files/model_image.png"
```

Comment:
can we use the smallest image possible, as it will contribute to the execution time?